Documents from last year's research

DavidMorin99 opened the pull request:
Pdf of Dr. Friedemann's Paper

DavidMorin99 committed:
Add files via upload

Pdf of Dr. Friedemann's Paper

sarboleda22 commented:
Looks good, just create a folder named `docs` and add it inside.
Also, name the file `surface_charges_research_dr_friedemann.pdf`

This way, you can create links in the wiki to direct people here.

DavidMorin99 committed:
Add files via upload

New folder titled "docs"
Renamed paper pdf file
Added file for semiconductor theory lecture from ENC Electronics class

DavidMorin99 committed:
Delete King1984 Phys Rev.pdf

DavidMorin99 committed:
Add files via upload

Added semiconductor lecture file with new name

DavidMorin99 committed:
Delete ENC_Chapter3_ELEC_2018.pptx

sarboleda22 commented:
Looks good :+1:
